### PR TITLE
1.3.1 release notes

### PR DESCRIPTION
**Pull request by brian-avery**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/boilerplates/notes/1.3.1.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Bug Fixes
+
+- **Fixed** an issue which caused the secret cleanup job to erroneously run during upgrades ([Issue 16873](https://github.com/istio/istio/issues/16873)).
+- **Fixed** an issue where the default configuration disabled Kubernetes Ingress support ([Issue 17148](https://github.com/istio/istio/issues/17148))
+- **Fixed** an issue with handling invalid UTF-8 characters in the Stackdriver logging adapter ([Issue 16966](https://github.com/istio/istio/issues/16966)).
```
Comment (sdake) [inline content/en/boilerplates/notes/1.3.1.md]: quote `UTF-8`

File: content/en/boilerplates/notes/1.3.1.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- **Fixed** Istio CNI sidecar initialization when IPv6 is disabled on the node ([Issue 15895](https://github.com/istio/istio/issues/15895)).
+- **Fixed** a regression affecting support of RS384 and RS512 algorithms in JWTs ([Issue 15380](https://github.com/istio/istio/issues/15380)).
+
+## Minor Enhancements
+
+- **Added** support for `.Values.global.priorityClassName` to the telemetry deployment ([Issue 16204](https://github.com/istio/istio/pull/16204)).
+- **Added** annotations for Datadog tracing that controls extra features in sidecars ([Issue 16594](https://github.com/istio/istio/pull/16594/files)).
+- **Added** the `pilot_xds_push_time` metric to report Pilot xDS push time ([Issue 17011](https://github.com/istio/istio/pull/17011)).
+- **Removed** time diff info in the proxy-status command ([Issue 16477](https://github.com/istio/istio/pull/16920)).
+- **Added** `istioctl experimental analyze` to support multi-resource analysis and validation ([Issue 17280](https://github.com/istio/istio/pull/17280)).
```
Comment (sdake) [inline content/en/boilerplates/notes/1.3.1.md]: can you swap this linie and the line above?  The bold tags should be grouped together.

**Comment by geeknoid** [comment]
You also need a blog post for the release announcement. And when merging this to the release-1.3 branch, you'll need to update the version number in data/args.yml.

**Comment by geeknoid** [comment]
https://github.com/istio/istio.io/pull/5060 just merged, which changes how we handle release notes.  We no longer use boilerplate text, instead the release note ends up as just a normal page under content/en/news/2019/*. There are many examples in the source base to follow.

Please rebase and update accordingly.

**Comment by brian-avery** [comment]
Thanks. Updating.

File: content/en/boilerplates/notes/1.3.1.md (hunk 2 above)
Comment (brian-avery) [reply content/en/boilerplates/notes/1.3.1.md]: done. Thanks.

File: content/en/news/2019/announcing-1.3.1/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title: Announcing Istio 1.3.1
+description: Istio 1.3.1 release announcement.
+publishdate: 2019-10-27
+attribution: The Istio Team
+subtitle: Minor Update
+release: 1.3.1
+aliases:
+    - /about/notes/1.3.1
```
Comment (geeknoid) [inline content/en/news/2019/announcing-1.3.1/index.md]: No need for the aliases, aliases are only needed when moving an old page, and this is a new page.

File: content/en/news/2019/announcing-1.3.1/index.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+- **Fixed** an issue with handling invalid `UTF-8` characters in the Stackdriver logging adapter ([Issue 16966](https://github.com/istio/istio/issues/16966)).
+- **Fixed** an issue which caused the `destination_service` label in HTTP metrics not to be set for `BlackHoleCluster` and `PassThroughCluster` ([Issue 16629](https://github.com/istio/istio/issues/16629)).
+- **Fixed** an issue with the `destination_service` label in the `istio_tcp_connections_closed_total` and `istio_tcp_connections_opened_total` metrics which caused them to not be set correctly ([Issue 17234](https://github.com/istio/istio/issues/17234)).
+- **Fixed** an Envoy crash introduced in Istio 1.2.4 ([Issue 16357](https://github.com/istio/istio/issues/16357)).
+- **Fixed** Istio CNI sidecar initialization when IPv6 is disabled on the node ([Issue 15895](https://github.com/istio/istio/issues/15895)).
+- **Fixed** a regression affecting support of RS384 and RS512 algorithms in JWTs ([Issue 15380](https://github.com/istio/istio/issues/15380)).
+
+## Minor Enhancements
```
Comment (geeknoid) [inline content/en/news/2019/announcing-1.3.1/index.md]:
```suggestion
## Minor enhancements
```

File: content/en/news/2019/announcing-1.3.1/index.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+aliases:
+    - /about/notes/1.3.1
+    - /blog/2019/announcing-1.3.1
+---
+
+This release includes bug fixes to improve robustness. This release note describes what’s different between Istio 1.3.0 and Istio 1.3.1.
+
+{{< relnote >}}
+
+## Bug Fixes
```
Comment (geeknoid) [inline content/en/news/2019/announcing-1.3.1/index.md]:
```suggestion
## Bug fixes
```

File: content/en/news/2019/announcing-1.3.1/index.md (hunk 3 above)
Comment (brian-avery) [reply content/en/news/2019/announcing-1.3.1/index.md]: fixed. Thanks.

File: content/en/news/2019/announcing-1.3.1/index.md (hunk 4 above)
Comment (brian-avery) [reply content/en/news/2019/announcing-1.3.1/index.md]: Fixed. Thanks.

File: content/en/news/2019/announcing-1.3.1/index.md (hunk 5 above)
Comment (brian-avery) [reply content/en/news/2019/announcing-1.3.1/index.md]: Fixed. Thanks.

**Comment by brian-avery** [comment]
Would that be https://github.com/istio/istio/pull/17333?

**Comment by rlenglet** [comment]
> Would that be istio/istio#17333?

We should only refer to issues, not PRs. I don't think there's an issue for this, AFAIK.

File: content/en/news/2019/announcing-1.3.1/index.md (hunk 3 above)
Comment (rlenglet) [reply content/en/news/2019/announcing-1.3.1/index.md]: Will the link be created automatically? This is the path that is referred by the release on GitHub.

File: content/en/news/2019/announcing-1.3.1/index.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+- **Fixed** an Envoy crash introduced in Istio 1.2.4 ([Issue 16357](https://github.com/istio/istio/issues/16357)).
+- **Fixed** Istio CNI sidecar initialization when IPv6 is disabled on the node ([Issue 15895](https://github.com/istio/istio/issues/15895)).
+- **Fixed** a regression affecting support of RS384 and RS512 algorithms in JWTs ([Issue 15380](https://github.com/istio/istio/issues/15380)).
+
+## Minor enhancements
+
+- **Added** support for `.Values.global.priorityClassName` to the telemetry deployment ([Issue 16204](https://github.com/istio/istio/pull/16204)).
+- **Added** annotations for Datadog tracing that controls extra features in sidecars ([Issue 16594](https://github.com/istio/istio/pull/16594/files)).
+- **Added** the `pilot_xds_push_time` metric to report Pilot xDS push time ([Issue 17011](https://github.com/istio/istio/pull/17011)).
+- **Added** `istioctl experimental analyze` to support multi-resource analysis and validation ([Issue 17280](https://github.com/istio/istio/pull/17280)).
```
Comment (mandarjog) [inline content/en/news/2019/announcing-1.3.1/index.md]: **Added** support for running metadata exchange and stats extensions in a WASM sandbox. Follow [these](https://istio.io/docs/ops/telemetry/in-proxy-service-telemetry/) instructions to try it out.

Comment (brian-avery) [reply content/en/news/2019/announcing-1.3.1/index.md]: Thanks. Will add. No issue we can link to here?

Comment (PiotrSikora) [reply content/en/news/2019/announcing-1.3.1/index.md]: Nit: "WebAssembly" and "Wasm" are the correct spellings, "WASM" isn't.

Comment (brian-avery) [reply content/en/news/2019/announcing-1.3.1/index.md]: Adopting WebAssembly since it's more verbose.

Comment (brian-avery) [reply content/en/news/2019/announcing-1.3.1/index.md]: Adopted WebAssembly since it's more verbose.

Comment (brian-avery) [reply content/en/news/2019/announcing-1.3.1/index.md]: Added. Thanks.

File: content/en/news/2019/announcing-1.3.1/index.md (hunk 3 above)
Comment (geeknoid) [reply content/en/news/2019/announcing-1.3.1/index.md]: No, the old page will not be created. We need to update the GitHub entry to point to the new location for the page.

File: content/en/news/2019/announcing-1.3.1/index.md (hunk 6 above)
Comment (mandarjog) [reply content/en/news/2019/announcing-1.3.1/index.md]: @gargnupur  can you update https://istio.io/docs/ops/telemetry/in-proxy-service-telemetry doc to include config options with wasm files?

File: content/en/news/2019/announcing-1.3.1/index.md (hunk 3 above)
Comment (rlenglet) [reply content/en/news/2019/announcing-1.3.1/index.md]: OK, so we need to update the release script to do so.

File: content/en/news/2019/announcing-1.3.1/index.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+---
+title: Announcing Istio 1.3.1
+description: Istio 1.3.1 release announcement.
+publishdate: 2019-10-27
```
Comment (PiotrSikora) [inline content/en/news/2019/announcing-1.3.1/index.md]: It's still September.

Comment (brian-avery) [reply content/en/news/2019/announcing-1.3.1/index.md]: Good catch. Thanks. Fixed.

**Pull request merged**